build-test.cmd

```diff
@@ -48,6 +51,8 @@ if /i "%1" == "checked"               (set __BuildType=Checked&set processedArgs
 if /i "%1" == "skipmanaged"           (set __SkipManaged=1&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
 if /i "%1" == "updateinvalidpackages" (set __UpdateInvalidPackagesArg=1&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
 if /i "%1" == "toolset_dir"           (set __ToolsetDir=%2&set __PassThroughArgs=%__PassThroughArgs% %2&set processedArgs=!processedArgs! %1 %2&shift&shift&goto Arg_Loop)
+if /i "%1" == "buildagainstpackages"  (set __BuildAgainstPackages=1&set __BuildAgainstPackagesArg=-BuildTestsAgainstPackages&shift&goto Arg_Loop)
+if /i "%1" == "runtimeid"             (set __RuntimeId=%2-x64&set processedArgs=!processedArgs! %1 %2&shift&shift&goto Arg_Loop)
```

Comment:
RuntimeID is for non-windows only, the reason I assumed x64 is because all of the runtime IDs we support are non-windows x64 builds. I.E. if the user wants to build tests for Ubuntu 14.04, they will be getting product binaries from Ubuntu.14.04-x64 packages. I could change this to have the user pass in the full runtimeID string exactly, though (Ubuntu.14.04-x64 instead of Ubuntu.14.04).